src/components/Capture/CountrySelector/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { h, FunctionComponent } from 'preact'
+import { useState } from 'preact/compat'
+import { omit } from 'lodash/fp'
+import { localised, useLocales } from '~locales'
+import Autocomplete from 'accessible-autocomplete/preact'
+
+import { allCountriesList, countryTranslations } from './countries'
+import { IconChevronDown } from '@onfido/castor-icons'
+import { getCountryFlagSrc } from '~supported-documents'
+import type { WithLocalisedProps } from '~types/hocs'
+import styles from '../../CountrySelector/style.scss'
+
+type CountryData = {
+  countryCode: string
+  labelKey?: string
+  isoAlpha3: string
+  label?: string
+}
+
+type Props = {
+  value: string | undefined
+  error: boolean
+  onChange: (value: string) => void
+} & WithLocalisedProps
+
+const getCountryOptionTemplate = (country: CountryData) => {
+  if (country) {
+    const countryCode = country.countryCode
+    const countryFlagSrc = getCountryFlagSrc(countryCode, 'square')
+    return `<i
+        role="presentation"
+        class="${styles.countryFlag}"
+        style="background-image: url(${countryFlagSrc})"></i>
+        <span class="${styles.countryLabel}">${country.label}</span>`
+  }
+  return ''
+}
+
+const options = allCountriesList.map((country: CountryData) => ({
+  ...omit('labelKey', country),
+  //@ts-ignore
+  label: countryTranslations[country.labelKey.replace('countriesList.', '')],
+}))
+
+const CountrySelector: FunctionComponent<Props> = ({
```

Comment:
Might be worth importing this component on-demand as it seems to be loading large files